src/main/java/edu/greenblitz/tobyDetermined/commands/telescopicArm/elbow/RotateToAngleRadians.java

```diff
@@ -31,17 +27,23 @@ public RotateToAngleRadians(double angle){
     @Override
     public void initialize() {
         super.initialize();
-        pidController = new ProfiledPIDController(PID.getKp(), PID.getKi(), PID.getKd(), CONSTRAINTS);
+        if(RobotMap.ROBOT_TYPE != Robot.RobotType.SIMULATION) {
```

Comment:
The robot type should not affect logic in the command layer, we can write a wrapper for ProfiledPIDController that will use the consts need depending on the robot type